src/ex5/server/clients/pokemon_client.js

```diff
@@ -0,0 +1,38 @@
+// The Pokemon Client (using axios) goes here
+
+const axios = require("axios");
+const popularPokemons = require("./popular_pokemons.js")
+
+class PokemonClient {
+  constructor() {
+    this.API_BASE = 'https://pokeapi.co/api/v2/pokemon';
+  }
+
+  async fetchPokemon(pokemonText) {
+    let pokemons = pokemonText.split(',').map( el => el.trim() );
+    pokemons = pokemons.filter(element => element.length > 0);
```

Comment:
love this handling of pokemon so some can fail and the rest still succeed

Reply:
It was added after your previous comment :)